pkg/processor/metricfrequencyprocessor/config.go

```diff
@@ -0,0 +1,18 @@
+package metricfrequencyprocessor
+
+import "go.opentelemetry.io/collector/config"
+
+// Config defines configuration for Source processor.
+type Config struct {
+	*config.ProcessorSettings `mapstructure:"-"`
+
+	MinPointAccumulationSeconds           int     `mapstructure:"min_point_accumulation_seconds"`
+	ConstantMetricsReportFrequencySeconds int64   `mapstructure:"constant_metrics_report_frequency_seconds"`
+	LowInfoMetricsReportFrequencySeconds  int     `mapstructure:"low_info_metrics_report_frequency_seconds"`
+	MaxReportFrequencySeconds             int     `mapstructure:"max_report_frequency_seconds"`
```

Comment:
`time.Duration` can be used for time types

Reply:
https://github.com/SumoLogic/sumologic-otel-collector/blob/main/pkg/extension/sumologicextension/config.go#L52
https://github.com/SumoLogic/sumologic-otel-collector/blob/main/pkg/extension/sumologicextension/extension.go#L71

Reply:
Handy, I forgot about it. I've switched to `time.Duration`. I'll update readme after we merge this PR.

Reply:
Can we update the README in this PR so that the change is "atomic"?